src/Compilers/Core/Portable/Collections/BitVector.cs

```diff
@@ -349,5 +350,15 @@ public static int WordsRequired(int capacity)
             if (capacity <= 0) return 0;
             return WordsForCapacity(capacity) + 1;
         }
+
+        internal string GetDebuggerDisplay()
+        {
+            var value = new char[_capacity];
+            for (int i = 0; i < _capacity; i++)
+            {
+                value[_capacity - i - 1] = this[i] ? '1' : '0';
```

Comment:
Why reverse the order?

Reply:
So the display value matches the binary representation of the int.